viewing_party/party.py

```diff
@@ -2,22 +2,178 @@
 
 def create_movie(title, genre, rating):
     pass
+    #make a dictionary with these keys: "title","genre","rating"
+    movie_dict = {}
+    movie_dict["title"]= title
+    movie_dict["genre"] = genre
+    movie_dict["rating"] = rating
+    
+    information = [title,genre,rating]
+    if None in information:
+        return None
+    else:
+        return movie_dict
+
+def add_to_watched(user_data, movie):
+    user_data["watched"] = [movie]
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"] = [movie]
+    return user_data 
+
+def watch_movie(user_data,title):
+    watched_movie = title
+    for movie in user_data["watchlist"]:
+        for value in movie.values():
+            if value == watched_movie:
+                user_data["watchlist"].remove(movie)
+                user_data["watched"].append(movie)
+
+    return(user_data)
 
 # -----------------------------------------
 # ------------- WAVE 2 --------------------
 # -----------------------------------------
+def get_watched_avg_rating(user_data):
+    ratings = []
+    ratings_sum = 0
+
+    if(len(user_data["watched"])) is 0:
+        ratings_average = 0.0
+    else:
+        for i in range(len(user_data["watched"])):
+            ratings.append((user_data["watched"][i]["rating"]))
+            i+=1

```

Comment:
good job! consider how we might make this for loop a little easier to understand at a glance.

We know `user_data["watched"` is a list, so why not use `for movie in user_data["watched]`. Now we can iterate over each movie and specify the rating key inside of it without our variable getting so long. It may look something like `ratings.append(movie["rating"])`

And remember, for loops will increment `i` on its own. We don't need to tell it to. It might actually mess it up and have it increment by 2 instead of just 1.